autosklearn/util/logging_.py

```diff
@@ -223,3 +223,4 @@ def serve_until_stopped(self) -> None:
 
             if self.event is not None and self.event.is_set():
                 break
+        print("Finished the process")
```

Comment:
Could you please drop this print?

Reply:
Sorry, removed. To catch this problem, we can add a pre-commit check that catches for prints. We will have to remove other prints, but we can make them warnings.warn.